q2cli/builtin/tools.py

```diff
@@ -404,7 +403,7 @@ def validate(path, level):
         result.validate(level)
     except qiime2.plugin.ValidationError as e:
         header = 'Result %s does not appear to be valid at level=%s:' % (
-                path, level)
+            path, level)
```

Comment:
Was this change necessary for the overall goal of the PR? If not, maybe we can undo the change (for the reasons given above). Sorry, don't mean to be pedantic, thanks for bearing with me!